### ISSUE TITLE
uint32_t redefined in type.h include (LeafNode_blink_nRF_test/src/main.c and leaf_sensor_node/src/main.c)

### ISSUE THREAD

**Issue by mescalan** [bunneydude/leaf_node#1]
type.h include in LeafNode_blink_nRF_test/src/main.c and leaf_sensor_node/src/main.c redefines the type declared in GNU C (GNU Tools for ARM Embedded Processors) stdint.h. 

In file included from c:\program files (x86)\gnu tools arm embedded\4.9 2014q4\lib\gcc\arm-none-eabi\4.9.3\include\stdint.h:9:0,
                 from ../cmsis/core_cm0plus.h:111,
                 from ../cmsis/LPC8xx.h:116,
                 from main.c:36:
c:\program files (x86)\gnu tools arm embedded\4.9 2014q4\arm-none-eabi\include\stdint.h:45:20: note: previous declaration of 'uint32_t' was here
 typedef __uint32_t uint32_t ;
                    ^
<builtin>: recipe for target 'main.o' failed
make: **\* [main.o] Error 1

**Comment by mescalan** [comment]
I just commented out the type.h include for now, but compiling takes a couple of extra Makefile changes

**Comment by bunneydude** [comment]
Yeah, it looks like the original LPC example code didn't include stdint and instead made it's own version in type.h. Going forward I think we'll be able to drop type.h and stick w/ stdint.h
